chart/templates/_helpers.yaml

```diff
@@ -703,14 +703,14 @@ Create the name of the cleanup service account to use
 {{- end -}}
 
 {{/*
-Set the default value for securityContext
-If no value is passed for securityContext or <node>.securityContext, defaults to global uid and gid.
+Set the default value for pod securityContext
+If no value is passed for securityContext.pod or <node>.securityContext.pod, defaults to global uid and gid.
```

Comment:
I think we want this order of precedence:

```
<node>.securityContexts.pod
<node>.securityContext
securityContexts.pod
securityContext (backcompat for deprecated config)
uid/gid
```

So I think this template still needs a little work, right? Also feel free to rename this `airflowPodSecurityContext` to make it more clear.

Reply:
You are right, I tried to adapt to your suggestion and renamed also the template.
Btw `airflowSecurityContextIds` was not used anywhere so the renamed version `localPodSecurityContextsIds` can probably be deleted.

Hope I got it right this time.

Reply:
Hi, I think airflowSecurityContextIds is used on workers-deployment to set the uid and gid during a chown. Not sure if has changed.